modules/nw-multus-create-network-apply.adoc

```diff
@@ -10,6 +10,7 @@
 
 * Install the OpenShift CLI (`oc`).
 * Log in as a user with `cluster-admin` privileges.
+* You are working in the namespace where the NAD is deployed.
```

Comment:
```suggestion
* You have installed the {oc-first}.
* You have logged in as a user with `cluster-admin` privileges.
* You have a network attachment definition (NAD) deployed in the namespace where you are working.
```
A few suggestions for prerequisites to maintain parallelism, expanding abbreviation, and using attribute.

Reply:
@shreyasiddhartha  A tiny nit here

"You are working in the namespace where you deployed the NAD."

Reply:
Makes sense given the context of the proc.